<!DOCTYPE html>
<html prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article#
" lang="en">
<head>
<meta charset="utf-8">
<meta name="description" content="materials for experiment">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>Lab_materials | Biohacker</title>
<link href="../../assets/css/bootstrap.min.css" rel="stylesheet" type="text/css">
<link href="../../assets/css/baguetteBox.min.css" rel="stylesheet" type="text/css">
<link href="../../assets/css/rst.css" rel="stylesheet" type="text/css">
<link href="../../assets/css/code.css" rel="stylesheet" type="text/css">
<link href="../../assets/css/theme.css" rel="stylesheet" type="text/css">
<link href="../../assets/css/custom.css" rel="stylesheet" type="text/css">
<link href="../../assets/css/ipython.min.css" rel="stylesheet" type="text/css">
<link href="../../assets/css/nikola_ipython.css" rel="stylesheet" type="text/css">
<meta name="theme-color" content="#5670d4">
<meta name="generator" content="Nikola (getnikola.com)">
<link rel="alternate" type="application/rss+xml" title="RSS" href="../../rss.xml">
<link rel="canonical" href="https://partrita.github.io/posts/lab_materials/">
<script type="text/x-mathjax-config">
 MathJax.Hub.Config({
     tex2jax: {
         inlineMath: [ ['$','$'], ["\\(","\\)"] ],
         displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
         processEscapes: true
     },
     displayAlign: 'center', // Change this to 'left' if you want left-aligned equations.
     "HTML-CSS": {
         styles: {'.MathJax_Display': {"margin": 0}}
     }
 });
 </script><!--[if lt IE 9]><script src="../../assets/js/html5.js"></script><![endif]--><link rel="stylesheet" type="text/css" href="../../assets/css/tipuesearch.css">
<link href="https://fonts.googleapis.com/css?family=Noto+Sans" rel="stylesheet">
<meta name="naver-site-verification" content="d38b92eb0fc0075616e5ff8008f707087bfee65e">
<!-- Global site tag (gtag.js) - Google Analytics --><script async src="https://www.googletagmanager.com/gtag/js?id=UA-111815051-1"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-111815051-1');
</script><!-- Google Tag Manager --><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-NHHK2BF');</script><meta name="author" content="Taeyoon Kim">
<link rel="prev" href="../python-model-fitting/" title="Model fitting by python" type="text/html">
<link rel="next" href="../ANOVA-python/" title="파이썬으로 분산분석(ANOVA)" type="text/html">
<meta property="og:site_name" content="Biohacker">
<meta property="og:title" content="Lab_materials">
<meta property="og:url" content="https://partrita.github.io/posts/lab_materials/">
<meta property="og:description" content="materials for experiment">
<meta property="og:type" content="article">
<meta property="article:published_time" content="2018-03-02T17:56:47+09:00">
<meta property="article:tag" content="materials">
<meta property="article:tag" content="protocols">
</head>
<body>
<a href="#content" class="sr-only sr-only-focusable">Skip to main content</a>

<!-- Menubar -->

<nav class="navbar navbar-expand-md static-top mb-4
navbar-dark
bg-dark
"><div class="container">
<!-- This keeps the margins nice -->
        <a class="navbar-brand" href="../../">

            <span id="blog-title">Biohacker</span>
        </a>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#bs-navbar" aria-controls="bs-navbar" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
        </button>

        <div class="collapse navbar-collapse" id="bs-navbar">
            <ul class="navbar-nav mr-auto">
<li class="nav-item">
<a href="../../pages/about/" class="nav-link">About</a>
                </li>
<li class="nav-item">
<a href="../../pages/resume/index.html" class="nav-link">Resume</a>
                </li>
<li class="nav-item">
<a href="../../pages/books/index.html" class="nav-link">Book</a>
                </li>
<li class="nav-item">
<a href="../../archive.html" class="nav-link">Archive</a>
                </li>
<li class="nav-item">
<a href="../../categories/" class="nav-link">Tags</a>
                </li>
<li class="nav-item">
<a href="../../rss.xml" class="nav-link">RSS feed</a>

                
            </li>
</ul>
<span class="navbar-form navbar-right">
<input type="text" id="tipue_search_input" class="form-control" placeholder="Search"></span>

            <ul class="navbar-nav navbar-right"></ul>
</div>
<!-- /.navbar-collapse -->
    </div>
<!-- /.container -->
</nav><!-- End of Menubar --><div class="container" id="content" role="main">
    <div class="body-content">
        <!--Body content-->
        
        
        
<article class="post-text h-entry hentry postpage" itemscope="itemscope" itemtype="http://schema.org/Article"><header><h1 class="p-name entry-title" itemprop="headline name"><a href="." class="u-url">Lab_materials</a></h1>

        <div class="metadata">
            <p class="byline author vcard p-author h-card"><span class="byline-name fn p-name" itemprop="author">
                    Taeyoon Kim
            </span></p>
            <p class="dateline">
            <a href="." rel="bookmark">
            <time class="published dt-published" datetime="2018-03-02T17:56:47+09:00" itemprop="datePublished" title="2018-03-02 17:56">2018-03-02 17:56</time></a>
            </p>
            

        </div>
        

    </header><div class="e-content entry-content" itemprop="articleBody text">
    <div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<p>실험하면서 자주 사용되는 시약의 정보를 모아두겠습니다.</p>
<hr>
<h2 id="Table-of-contents">Table of contents<a class="anchor-link" href="#Table-of-contents">¶</a>
</h2>
<ol>
<li><a href="#media">Media</a></li>
<li><a href="#antibiotics">Antibiotics</a></li>
<li><a href="#enzymes">Enzymes</a></li>
<li><a href="#buffers">Buffers</a></li>
<li><a href="#others">Other common chemicals</a></li>
</ol>
<hr>
<p>This category from <a href="https://openwetware.org/wiki/Materials">OpenWetWare</a></p>
<hr>
</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<div id="media"></div>
# 1. Media
--------

<h3 id="1.1.-Defined-Media">1.1. Defined Media<a class="anchor-link" href="#1.1.-Defined-Media">¶</a>
</h3>
<h4 id="Synthetic-Complete-(SC)-Medium-aka-(CSM,-CM).-Defined-yeast-medium">Synthetic Complete (SC) Medium aka (CSM, CM). Defined yeast medium<a class="anchor-link" href="#Synthetic-Complete-(SC)-Medium-aka-(CSM,-CM).-Defined-yeast-medium">¶</a>
</h4>
<ul>
<li>Adenine sulfate 20 mg/L</li>
<li>Uracil 20 mg/L</li>
<li>L-Tryptophan 20 mg/L</li>
<li>L-Histidine-HC1 20 mg/L</li>
<li>L-Arginine-HC1 20 mg/L</li>
<li>L-Methionine 20 mg/L</li>
<li>L-Tyrosine 30 mg/L</li>
<li>L-Leucine 60 mg/L</li>
<li>L-Isoleucine 30 mg/L</li>
<li>L-Lysine-HC1 30 mg/L</li>
<li>L-Phenylalanine 50 mg/L</li>
<li>L-Glutamic acid 100 mg/L</li>
<li>L-Aspartic acid 100 mg/L</li>
<li>L-Valine 150 mg/L</li>
<li>L-Threonine 200 mg/L</li>
<li>L-Serine 400 mg/L</li>
</ul>
<h3 id="1.2.-Standard-Media">1.2. Standard Media<a class="anchor-link" href="#1.2.-Standard-Media">¶</a>
</h3>
<h4 id="Luria-Bertani-Medium-(aka-L-Broth-or-LB-Medium)">Luria-Bertani Medium (aka L-Broth or LB Medium)<a class="anchor-link" href="#Luria-Bertani-Medium-(aka-L-Broth-or-LB-Medium)">¶</a>
</h4>
<p>LB is a standard growth medium for a variety of bacteria and conditions.</p>
<h5 id="Ingredients-for-1L">Ingredients for 1L<a class="anchor-link" href="#Ingredients-for-1L">¶</a>
</h5>
<ul>
<li>10 g Bacto-tryptone</li>
<li>5 g yeast extract</li>
<li>10 g NaCl</li>
</ul>
<p>For plates also add</p>
<ul>
<li>15g agar</li>
</ul>
<blockquote>
<p>Note: There are two formulations of LB, Miller and Lennox, that differ in the amount of NaCl. Lennox has less salt, only 5 g/L. The Qiagen miniprep kit recommends LB with 10 g NaCl for highest plasmid yields.</p>
</blockquote>
<h5 id="Protocol">Protocol<a class="anchor-link" href="#Protocol">¶</a>
</h5>
<ol>
<li>Mix dry ingredients and add distilled water up to 1 Liter</li>
<li>Pour into 2 L flask (or greater)</li>
<li>Autoclave (liquid cycle) 250°F, 22psi, 30 minutes</li>
</ol>
<blockquote>
<p>Notes: We do not pH medium when we make it on the fly. However, if it is really important, pH the medium to 7.0 with 5M NaOH (~200µL). Check with pH paper</p>
</blockquote>
<h5 id="For-plates">For plates<a class="anchor-link" href="#For-plates">¶</a>
</h5>
<ol>
<li>Mix dry ingredients and add distilled water up to less than 1 Liter</li>
<li>After dissolved, add agar</li>
<li>Pour into 2 L flask (or greater)</li>
<li>Autoclave (liquid cycle): 250°F, 22psi, 30 minutes</li>
<li>Cool media down to 60-55C (uncomfortable but not painful)</li>
<li>Add appropriate antibiotic</li>
<li>Pour the solution on sterile plates</li>
<li>Let solidify about 1 h, wrap, label (name, date, additive) keep at 4°C</li>
</ol>
<hr>
</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<div id="anitibiotics"></div>
# 2. Antibiotics

<hr>
<h3 id="Kanamycin">Kanamycin<a class="anchor-link" href="#Kanamycin">¶</a>
</h3>
<h4 id="Mode-of-Action">Mode of Action<a class="anchor-link" href="#Mode-of-Action">¶</a>
</h4>
<p>Bacteriocidal. Diffuses through the porinchannels in the outer membrane of gram-negative bacteria. Interacts with at least three ribosomal proteins, inhibiting protein synthesis and increasing translation errors.</p>
<h4 id="Working-Concentration-and-Stock-Solution">Working Concentration and Stock Solution<a class="anchor-link" href="#Working-Concentration-and-Stock-Solution">¶</a>
</h4>
<p>Working concentration is 50 μg/ml, or 25 μg/ml for low-copy plasmids. Stock solution is 35 mg/ml in water (kanamycin is insoluble in 50% alcohol).</p>
<h3 id="Ampicillin">Ampicillin<a class="anchor-link" href="#Ampicillin">¶</a>
</h3>
<h4 id="Mode-of-Action">Mode of Action<a class="anchor-link" href="#Mode-of-Action">¶</a>
</h4>
<p>Inhibits the formation of cross-links in the peptidoglycan layer (which provides rigidity to the cell wall). Most effective against cells in log phase growth (since this is when new cross-links are being formed), and has little effect on cells in stationary phase.</p>
<h4 id="Stock-Solution">Stock Solution<a class="anchor-link" href="#Stock-Solution">¶</a>
</h4>
<p>Typical concentrations of ampicillin are 50 μg/mL for low copy plasmids and 100 μg/mL for high copy plasmids. Stock solutions are typically at 100 mg/ml, so that 1 ml of antibiotic can be added to 1 liter of broth or agar. Stock solutions made in 50% alcohol remain liquid at -20°C and are easy to pipet. Cool agar to 55°C or below prior to adding antibiotic.</p>
<p>Ampicillin available from Sigma A-9518 (Ampicillin sodium salt), FW 371.39. To make 100ml of 100 mg/ml stock solution, dissolve 10 g of ampicillin in 50 ml of water and 50 ml of 100% ethanol.</p>
<h4 id="Stability">Stability<a class="anchor-link" href="#Stability">¶</a>
</h4>
<p>Culture plates with ampicillin can be stored at 2-8°C for up to two weeks. (In our experience, plates are usable for 2-3 months when stored at 4°C and bagged to prevent evaporation.)<br>
Stock solutions may be stored at 2-8°C for up to 3 weeks. For long term storage (4-6 months), stock solutions should be stored at -20°C. At 37°C in culture, ampicillin is stable up to 3 days. <a href="https://www.sigmaaldrich.com/sigma/product%20information%20sheet/a0166pis.pdf">Sigma reference</a></p>
<h3 id="Carbenicillin">Carbenicillin<a class="anchor-link" href="#Carbenicillin">¶</a>
</h3>
<p>Carbenicillin is used to select for cells bearing the beta-lactamase gene bla or ampR. Beta-lactamase also neutralizes ampicillin, but due to ampicillin's lower effectiveness, carbenicillin tends to give fewer satellite colonies and makes for more reliable cloning. Ampicillin is more widely used, though, due to carbenicillin's high cost ($75/g compared to ampicillin's $10/g (Sigma)).</p>
<h3 id="Tetracycline">Tetracycline<a class="anchor-link" href="#Tetracycline">¶</a>
</h3>
<p>Tetracyclines (there are many) enter the cell by diffusion through OmpF porins. It inhibits growth by preventing codon-anticodon interactions during translation.</p>
<h4 id="Working-concentration-and-Stock-Solutions">Working concentration and Stock Solutions<a class="anchor-link" href="#Working-concentration-and-Stock-Solutions">¶</a>
</h4>
<p>Using a plasmid with copy number 15-20, 20μg/ml tet has proved a successful selector. The reduction in growth rate, described above was significant.
The RajBhandary lab uses 5μg/ml and see a minimal reduction in growth rate and never lose plasmid from their cells.
Stock solution is 15 mg/ml in 50% ethanol (note: using 95-100% ethanol is much better for dissolving stock powder). Add 1 ml per liter of cooled broth or agar. Solutions and plates are light sensitive.
Tetracycline is incompatible with (chelated by) magnesium ions, and thus unsuitable for use in minimal media</p>
<h3 id="Streptomycin">Streptomycin<a class="anchor-link" href="#Streptomycin">¶</a>
</h3>
<h4 id="Mode-of-Action">Mode of Action<a class="anchor-link" href="#Mode-of-Action">¶</a>
</h4>
<p>Inhibits protein synthesis; binds to 30S ribosomal subunit. The rpsL50 gene mutation conveys streptomycin resistance. Resistance is recessive, so the presence of a wild type rpsL gene will convey sensitivity. This allows counter-selection against the presence of the wt rpsL gene. Mutations of wild type genes to convey resistance are common.</p>
<h4 id="Working-concentration-and-Stock-Solutions">Working concentration and Stock Solutions<a class="anchor-link" href="#Working-concentration-and-Stock-Solutions">¶</a>
</h4>
<p>100 mg/mL stock solution in water. It is barely soluble in ethanol so you must dissolve it in water. Add 1 mL per liter of culture.</p>
<h3 id="Chloramphenicol">Chloramphenicol<a class="anchor-link" href="#Chloramphenicol">¶</a>
</h3>
<p>Chloramphenicol is a bacteriostatic agent that binds to the 50S ribosomal subunit and inhibits ribosomal peptide bond formation. It is sometimes used as a way of "amplifying" plasmid production by shutting down protein synthesis in cultures, while allowing plasmid replication to continue.</p>
<h4 id="Working-Concentrations-and-Stock-Solutions">Working Concentrations and Stock Solutions<a class="anchor-link" href="#Working-Concentrations-and-Stock-Solutions">¶</a>
</h4>
<p>High copy plasmids allow 35 μg/ml concentration. Low-copy plasmids like bacterial artificial chromosomes allow 12.5 μg/ml. Stock solutions can be made at 35 mg/ml in ethanol, kept at -20C.</p>
<h3 id="Neomycin">Neomycin<a class="anchor-link" href="#Neomycin">¶</a>
</h3>
<p>binds to ribosomal components and inhibits protein synthesis. target organism: bacteria, mammalian cells</p>
<h4 id="Working-Concentrations-and-Stock-Solutions">Working Concentrations and Stock Solutions<a class="anchor-link" href="#Working-Concentrations-and-Stock-Solutions">¶</a>
</h4>
<p>stock solution: 10mg/ml in water
working solution: 10-50µg/ml (stable at RT for 5days)</p>
<hr>
</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<div id="enzymes"></div>
<h2 id="3.-Enzymes">3. Enzymes<a class="anchor-link" href="#3.-Enzymes">¶</a>
</h2>
<hr>
<h3 id="3.1-Phosphatases">3.1 Phosphatases<a class="anchor-link" href="#3.1-Phosphatases">¶</a>
</h3>
<h4 id="Antarctic-Phosphatase">Antarctic Phosphatase<a class="anchor-link" href="#Antarctic-Phosphatase">¶</a>
</h4>
<p>This enzyme is preferable to either Calf Intestinal Phosphatase or Shrimp Alkaline Phosphatase because it is more easily heat-inactivated. After phosphatase treatment of a vector, any remaining enzymatic activity would remove 5' phosphates from the insert in a subsequent ligation reaction, preventing a successful reaction. Thus, it is critical that the phosphatase be inactivated, purified away, or digested with Proteinase K.</p>
<h3 id="3.2.-DNA-polymerases">3.2. DNA polymerases<a class="anchor-link" href="#3.2.-DNA-polymerases">¶</a>
</h3>
<h4 id="Pfu">Pfu<a class="anchor-link" href="#Pfu">¶</a>
</h4>
<p>Pfu has proof reading activity which makes it more accurate than for example Taq.
Pfu does not exhibit 5' -&gt; 3' exonuclease activity, which prevents degradation of the primers in mutagenesis methods based on ligation of primers or elongated primers.
Pfu is slower than Taq and one should calculate 2 min per 1000 bp (1 min for Taq)</p>
<h4 id="T4-DNA-Polymerase">T4 DNA Polymerase<a class="anchor-link" href="#T4-DNA-Polymerase">¶</a>
</h4>
<p>LIC quality T4 DNA Polymerase can be used to "chewback" 3' ends for use in Ligation Independent Cloning (LIC).</p>
<h3 id="3.3.-DNA-ligases">3.3. DNA ligases<a class="anchor-link" href="#3.3.-DNA-ligases">¶</a>
</h3>
<p>DNA ligase is a specific type of enzyme, a ligase, that facilitates the joining of DNA strands together by catalyzing the formation of a phosphodiester bond.</p>
<h4 id="3.4.-RNA-Polymerases">3.4. RNA Polymerases<a class="anchor-link" href="#3.4.-RNA-Polymerases">¶</a>
</h4>
<h4 id="3.5.-Restriction-enzymes">3.5. Restriction enzymes<a class="anchor-link" href="#3.5.-Restriction-enzymes">¶</a>
</h4>
<p>Restriction enzymes are used for performing a restriction digest.</p>
<p>Comprehensive information can be found in REBASE. Neeraj Dhar's good cutter list is a list of enzymes which are particularly reliable and straightforwards.</p>
<p>This is another source of info on restricition enzymes. It's by the same people who provide REBASE, but in a different format:</p>
<h3 id="3.6.-Other">3.6. Other<a class="anchor-link" href="#3.6.-Other">¶</a>
</h3>
<h4 id="Lysozyme">Lysozyme<a class="anchor-link" href="#Lysozyme">¶</a>
</h4>
<p>Breaks down cell walls by catalyzing the hydrolysis of cell wall peptidoglycans.</p>
<hr>
</div>
</div>
</div>
<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
</div>
<h3 id="4.-Buffers">4. Buffers<a class="anchor-link" href="#4.-Buffers">¶</a>
</h3>
<hr>
<h3 id="TAE">TAE<a class="anchor-link" href="#TAE">¶</a>
</h3>
<p>TAE is a commonly used buffer for making and running DNA agarose gels. It offers a few advantages and disadvantages compared to TBE buffer:</p>
<ul>
<li>TAE buffer provides optimal resolution of fragments &gt;4 kb in length, while TBE provides better resolution for 0.1 to 3 kb fragments.</li>
<li>TAE is significantly cheaper to make</li>
<li>TAE stocks can be 50X concentrated and therefore take up less space than 10X concentrated TBE stock</li>
<li>TBE offers a higher resolution and has a higher buffering capacity at greater temperatures induced by relatively higher voltages</li>
<li>TBE can negatively influence the yield of DNA after recovery from a gel when using glass-based protocols. Yield does not seem to be influenced when using silica-based methods.</li>
<li>The bromide ion in TBE is a powerful inhibitor of enzyme activity. This can prevent degradation of nucleic acid, but can also interfere with subsequent experiments like cloning DNA into a vector.</li>
</ul>
<p>Ingredients for one litre 50X stock</p>
<ul>
<li>Tris-base: 242 g</li>
<li>Acetate (100% acetic acid): 57.1 ml</li>
<li>EDTA: 100 ml 0.5M sodium EDTA</li>
</ul>
<p>Add dH2O up to one litre.</p>
<p>To make 1x TAE from 50X TAE stock, dilute 20ml of stock into 980 ml of deionised water.</p>
<h3 id="PBS">PBS<a class="anchor-link" href="#PBS">¶</a>
</h3>
<p>PBS stands for phosphate buffered saline and is often used because it is isotonic to cells.</p>
<h4 id="Recipe">Recipe<a class="anchor-link" href="#Recipe">¶</a>
</h4>
<p>1x PBS is 137 mM NaCl, 12 mM Phosphate, 2.7 mM KCl, pH 7.4<br>
To make 10x PBS,</p>
<ol>
<li>Combine the following:<ul>
<li>80 g NaCl</li>
<li>2 g KCl</li>
<li>14.4 g Na2HPO4 (dibasic anhydrous) OR 18.1 g Na2HPO4 · 2H2O (dibasic dihydrate) OR 27.2 g Na2HPO4 · 7H2O (dibasic heptahydrate)</li>
<li>2.4 g KH2PO4 (monobasic anhydrous)</li>
<li>800 mL distilled H2O</li>
</ul>
</li>
<li>Adjust pH to 7.4 with HCl</li>
<li>Add H2O to 1L</li>
<li>Autoclave for 20 minutes on liquid cycle. Store at RT.</li>
</ol>
<h3 id="PBST">PBST<a class="anchor-link" href="#PBST">¶</a>
</h3>
<p>For some applications (washing) one adds Tween 20 PBS which makes PBST</p>
<ul>
<li>Usual concentrations of Tween 20 in PBST are 0.1 or 0.05%.</li>
<li>Usual concentration of Triton X-100 in TBST is 0.1-0.3%.</li>
</ul>
<h3 id="TE">TE<a class="anchor-link" href="#TE">¶</a>
</h3>
<p>TB buffer is used to make chemically competent cells by the Inoue method Preparing chemically competent cells (Inoue)</p>
<p>Preparation: For a 1 liter solution add</p>
<ul>
<li>250 mM potassium chloride (18.65 g)</li>
<li>15 mM calcium chloride (2.2 g)</li>
<li>10 mM PIPES (20 ml of an 0.5 M solution)</li>
</ul>
<p>to 800 ml of DI water. Adjust pH to 6.7 with 1M KOH.</p>
<p>Dissolve</p>
<ul>
<li>55 mM manganese chloride (10.88 g)</li>
</ul>
<p>in 100 ml of DI water, add gradually to the solution of the remaining components. Bring solution to 1 liter. The pH of the solution will fall, which is expected.</p>
<blockquote>
<p>Do not attempt to adjust the pH. Adding base after adding manganese will precipitate a yellow/brown hydroxide.</p>
</blockquote>
<p>Sterile filter with a pre-washed 0.22 μm filter and store indefinitely at 4 degrees.</p>
<h3 id="NEB-buffers">NEB buffers<a class="anchor-link" href="#NEB-buffers">¶</a>
</h3>
<ul>
<li>
<p>NEBuffer 1 (yellow)</p>
<ul>
<li>10 mM Bis Tris Propane-HCl, 10 mM MgCl2, 1 mM DTT (pH 7.0 at 25°C).</li>
</ul>
</li>
<li>
<p>NEBuffer 2 (blue)</p>
<ul>
<li>10 mM Tris-HCl, 10 mM MgCl2, 50 mM NaCl, 1 mM DTT (pH 7.9 at 25°C).</li>
</ul>
</li>
<li>
<p>NEBuffer 3 (red)</p>
<ul>
<li>50 mM Tris-HCl, 10 mM MgCl2, 100 mM NaCl, 1 mM DTT (pH 7.9 at 25°C).</li>
</ul>
</li>
<li>
<p>NEBuffer 4 (green)</p>
<ul>
<li>20 mM Tris-acetate, 10 mM magnesium acetate, 50 mM potassium acetate, 1 mM DTT (pH 7.9 at 25°C).</li>
</ul>
</li>
<li>
<p>T4 DNA Ligase Buffer</p>
<ul>
<li>50 mM Tris-HCl, 10 mM MgCl2, 1 mM ATP, 10 mM Dithiothreitol (pH 7.5 at 25°C).</li>
</ul>
</li>
<li>
<p>NEBuffer EcoRI</p>
<ul>
<li>100 mM Tris-HCl, 50 mM NaCl, 10 mM MgCl2, 0.025 % Triton X-100 (pH 7.5 at 25°C).</li>
</ul>
</li>
</ul>
<h3 id="SDS-sample-buffer">SDS sample buffer<a class="anchor-link" href="#SDS-sample-buffer">¶</a>
</h3>
<p>For preparation and loading of protein samples onto a gel for SDS-PAGE analysis (Western blot/protein blot).</p>
<h4 id="Preparation:">Preparation:<a class="anchor-link" href="#Preparation:">¶</a>
</h4>
<p>SDS contained in the sample buffer makes proteins negatively charged proportionally to their length. 2-mercapto-ethanol/DTT breaks disulphide bonds.</p>
<h4 id="Loading:">Loading:<a class="anchor-link" href="#Loading:">¶</a>
</h4>
<p>glycerol makes the sample buffer more dense than the surrounding running buffer of the protein gel, enabling easy loading into the gel pockets.</p>
<h4 id="To-make-10-mL-of-4x-stock">To make 10 mL of 4x stock<a class="anchor-link" href="#To-make-10-mL-of-4x-stock">¶</a>
</h4>
<ul>
<li>2.0 ml 1M Tris-HCl pH 6.8</li>
<li>0.8 g SDS</li>
<li>4.0 ml 100% glycerol</li>
<li>0.4 ml 14.7 M β-mercaptoethanol</li>
<li>1.0 ml 0.5 M EDTA</li>
<li>8 mg bromophenol Blue</li>
</ul>
<h4 id="Final-concentrations-(1x)">Final concentrations (1x)<a class="anchor-link" href="#Final-concentrations-(1x)">¶</a>
</h4>
<ul>
<li>50 mM Tris-HCl pH 6.8</li>
<li>2% SDS</li>
<li>10% glycerol</li>
<li>1% β-mercaptoethanol</li>
<li>12.5 mM EDTA</li>
<li>0.02 % bromophenol blue</li>
</ul>
<h3 id="TBS">TBS<a class="anchor-link" href="#TBS">¶</a>
</h3>
<p>TBS stands for tris-buffered saline and is used in many protocols for washing Endconzentration in 1x TBS will be 20mM Tris and 150mM NaCl</p>
<h4 id="5xTBS-for-1-liter-with-stock-solutions:">5xTBS for 1 liter with stock solutions:<a class="anchor-link" href="#5xTBS-for-1-liter-with-stock-solutions:">¶</a>
</h4>
<ul>
<li>100 ml 1M Tris-HCl pH 7.5</li>
<li>150 ml 5M NaCl</li>
<li>750 ml ddH2O</li>
</ul>
<p>Prepare and sterilize filter. Store at 4°C. After diluting the 5x to 1x filter again</p>
<h3 id="TBST">TBST<a class="anchor-link" href="#TBST">¶</a>
</h3>
<p>TBST: For some applications one adds Tween 20 or Triton X-100 to TBS which makes TBST (people use the name TBST for both, probably it's lab dependent.</p>
<ul>
<li>Usual concentrations of Tween 20 in TBST are 0.1 or 0.05%.</li>
<li>Usual concentration of Triton X-100 in TBST is 0.1%.</li>
</ul>
<h3 id="Killing-Buffer">Killing Buffer<a class="anchor-link" href="#Killing-Buffer">¶</a>
</h3>
<p>Killing Buffer is used to kill bacterial cells. Usually one would use cold Killing buffer in a 50:50 ratio to harvest cells in liquid medium. The buffer contains NaN3 and kills the cells quickly. Fast killing and down cooling is important for example to stop RNases if you want to isolate RNA from that specific time point of cell growth.</p>
<p>Endconzentration will be 20mM Tris and 5mM MgCl2 and 20mM NaN3</p>
<h4 id="Killing-Buffer-for-1-liter:">Killing Buffer for 1 liter:<a class="anchor-link" href="#Killing-Buffer-for-1-liter:">¶</a>
</h4>
<ul>
<li>2 ml 1M Tris-HCl pH 7.5</li>
<li>0.5 ml 1M MgCl2</li>
<li>1.3 g NaN3</li>
<li>997.5 ml ddH2O</li>
</ul>
<p>Store at 4°C.</p>
<blockquote>
<p>Note that NaN3 is highly toxic and should be handled accordingly.</p>
</blockquote>
<hr>
</div>
</div>

<div class="cell border-box-sizing text_cell rendered">
<div class="prompt input_prompt">
</div>
<div class="inner_cell">
<div class="text_cell_render border-box-sizing rendered_html">
<div id="others"></div>
<h2 id="5.-Other-common-chemicals">5. Other common chemicals<a class="anchor-link" href="#5.-Other-common-chemicals">¶</a>
</h2>
<hr>
<h3 id="Ethidium-bromide">Ethidium bromide<a class="anchor-link" href="#Ethidium-bromide">¶</a>
</h3>
<p>Ethidium Bromide, often abbreviated EtBr. Intercalating agent that inserts itself between DNA base pairs Fluorescent under UV light</p>
<blockquote>
<p>Caution: Carcinogen/Mutagen</p>
</blockquote>
<h3 id="IPTG">IPTG<a class="anchor-link" href="#IPTG">¶</a>
</h3>
<p>Isopropyl-beta-D-thiogalactopyranoside
Induces transcription from promoters regulated by LacI repressor.
Molecular weight is 238.31 g/mol. The chemical formula is here.</p>
<ol>
<li>Dissolve 1g in 4196 μL deionized water to make 1M solution.</li>
<li>Filter sterilize with syringe and 0.22μm filter</li>
</ol>
<p>Alternatively</p>
<ol>
<li>Dissolve 238 mg IPTG (sigma I-6758) in 10 ml DW for (100mM stock)</li>
<li>store in 1 ml aliquotes at -20°C</li>
</ol>
<p>Generally a 1mM solution is an effective amount to induce the pLac promoter region.</p>
<h3 id="EDTA">EDTA<a class="anchor-link" href="#EDTA">¶</a>
</h3>
<p>EDTA stands for ethylene-diamine-tetraacetic acid. It chelates divalent cations and is therefore used in many buffers. Its relative EGTA has a higher affinity for calcium than for magnesium ions.
<img alt="edta" src="https://s3-us-west-2.amazonaws.com/oww-files-thumb/b/b9/EDTA.png/200px-EDTA.png"></p>
<h4 id="0.5-M-EDTA-stock">0.5 M EDTA stock<a class="anchor-link" href="#0.5-M-EDTA-stock">¶</a>
</h4>
<ul>
<li>18.61 g EDTA (Sodium Salt)</li>
<li>dH2O to 90 ml</li>
<li>adjust pH to 7.0</li>
<li>adjust volume to 100 ml</li>
</ul>
<h4 id="0.5M-500ml-pH-8.0-with-NaOH-pre-calculated">0.5M 500ml pH 8.0 with NaOH pre-calculated<a class="anchor-link" href="#0.5M-500ml-pH-8.0-with-NaOH-pre-calculated">¶</a>
</h4>
<ul>
<li>93.05g of Na2.EDTA (FW 372.2)</li>
<li>10.14g of NaOH (FW 40)</li>
<li>500 ml dH2O</li>
</ul>
<h3 id="Phenol">Phenol<a class="anchor-link" href="#Phenol">¶</a>
</h3>
<p>Phenol is an important chemical in biological research. It is used primarily for isolation and purification of DNA and RNA. Pure phenol is solid at room temperature. Almost all biochemical uses of phenol use a water saturated phenol solution, or a mixture of phenol in chloroform.</p>
<h4 id="Safety">Safety<a class="anchor-link" href="#Safety">¶</a>
</h4>
<p>Phenol is caustic and causes nasty chemical burns which are slow to heal. It is also a systemic poison which can be rapidly absorbed through the skin.</p>
<h3 id="Glucose">Glucose<a class="anchor-link" href="#Glucose">¶</a>
</h3>
<p>The molecular weight of anhydrous glucose is 180.16 g/mol.
To make 1M glucose, dissolve 18g in 100mL H2O.
1M glucose = 18% glucose</p>
<h3 id="PMSF">PMSF<a class="anchor-link" href="#PMSF">¶</a>
</h3>
<p>PMSF stands for Phenylmethylsulfonyl fluoride. It is used as an irreversible inhibitor of serine protease activity for example in protein purification or co-immunoprecipitation. It has no effect on other proteases that might be present. It can also inactivate some non-proteolytic enzymes that contain serine in the active site.</p>
<h4 id="Use">Use<a class="anchor-link" href="#Use">¶</a>
</h4>
<p>PMSF is not soluble in water, dissolve in isopropanol, ethanol or methanol. 200 mM stock solution (0.035 g/mL) is stable for months at 4°C, although storage at -20 is probably better. A common final concentration of PMSF is 1mM. PMSF is extremely toxic, and preparation of solutions should be done very carefully. PMSF is not stable in alkaline conditions, or in water. For disposal, add alkali in water and wait to dispose.</p>
<h3 id="Safety">Safety<a class="anchor-link" href="#Safety">¶</a>
</h3>
<p>PMSF is extremely poisonous and should be handled carefully.</p>
<h3 id="Glycerol">Glycerol<a class="anchor-link" href="#Glycerol">¶</a>
</h3>
<p><img alt="" src="https://s3-us-west-2.amazonaws.com/oww-files-thumb/6/6d/Glycerol_chemical_formula.svg/300px-Glycerol_chemical_formula.svg.png">
Glycerol is an alcohol often used in the lab to avoid freezing and increase the density of liquids. Its standardised IUPAC name is propane-1,2,3-triol. It is also often referred to as glycerine or glycerin.</p>
<h4 id="Density">Density<a class="anchor-link" href="#Density">¶</a>
</h4>
<p>Glycerol is used in many loading buffers to increase the density of the solution which enables good loading into already liquid filled wells of agarose and polyacrylamide gels. See agarose gel loading buffer and SDS sample buffer.</p>
<h4 id="Freezing">Freezing<a class="anchor-link" href="#Freezing">¶</a>
</h4>
<p>Glycerol is also used to decrease the freezing point of protein solution which are easily degraded by repeated freeze/thaw cycles. A 50% (w/v) glycerol/water solution freezes at -25ºC and can thus be stored in a -20ºC freezer. Note that glycerol with 1.26g/cm3 is much denser than water and, therefore, a 50% (v/v) solution will be very different from a 50% (w/v) solution. For example, restriction enzymes are typically stored in a glycerol/water solution.</p>
<h3 id="Ficoll">Ficoll<a class="anchor-link" href="#Ficoll">¶</a>
</h3>
<p>Ficoll™ is a sucrose polymer commonly used to adjust the density and viscosity of solutions in the lab. It is employed in density gradients and loading buffers.</p>
<h4 id="Use">Use<a class="anchor-link" href="#Use">¶</a>
</h4>
<p>One of the most common applications for Ficoll is as a density gradient for the isolation of eukaryotic cells, organelles, bacterial cells, and pancreatic islets.</p>
<h4 id="Properties">Properties<a class="anchor-link" href="#Properties">¶</a>
</h4>
<p>Ficoll is an uncharged, highly branched polymer formed by the co-polymerisation of sucrose and epichlorohydrin. Due to its many hydroxyl groups, Ficoll is highly soluble in water. Densities of up to 1.2 g/ml can be attained.</p>
<h4 id="Stability-&amp;-storage">Stability &amp; storage<a class="anchor-link" href="#Stability-&amp;-storage">¶</a>
</h4>
<p>Since Ficoll does not contain any ionized groups, the molecule does not readily react under physiological conditions. It is stable in alkaline and neutral solutions, but is rapidly hydrolysed in solution at pH 3, especially at high temperature. However, Ficoll can be autoclaving at 110ºC for 30 minutes in neutral solutions. Strong oxidizing and reducing agents should be avoided. Shipping and storage are at ambient temperatures.</p>
<h3 id="DNA-dyes">DNA dyes<a class="anchor-link" href="#DNA-dyes">¶</a>
</h3>
<p>DNA dyes stain deoxyribonucleic acid for laboratory purposes such as detection and quantification. Many DNA dyes also bind to RNA and could be more broadly described as nucleic acid stains. Common dyes included ethidium bromide (EtBr), esp. for agarose gel electrophoresis of DNA, and DAPI for staining the cell nucleus in fluorescent microscopy.</p>
<h4 id="Dyes-for-DNA-in-agarose-gel-electrophoresis">Dyes for DNA in agarose gel electrophoresis<a class="anchor-link" href="#Dyes-for-DNA-in-agarose-gel-electrophoresis">¶</a>
</h4>
<ul>
<li>Ethidium bromide : traditional DNA stain.</li>
<li>SYBR Gold : 10x more sensitive than EtBr</li>
<li>SYBR Green : 20x more sensitive than EtBr</li>
<li>SYBR Safe : sensitivity similar to EtBr</li>
</ul>
<h3 id="DNTPs">DNTPs<a class="anchor-link" href="#DNTPs">¶</a>
</h3>
<p>dNTP's or deoxynucleosides are the monomers that DNA polymerase uses to form DNA.</p>
<h3 id="ProteinA-Sepharose-Beads">ProteinA Sepharose Beads<a class="anchor-link" href="#ProteinA-Sepharose-Beads">¶</a>
</h3>
<p>P3391 ProteinA-Sepharose beads from Staphylococcus aureus. Lyophilized powder</p>
<h3 id="Tween-20">Tween 20<a class="anchor-link" href="#Tween-20">¶</a>
</h3>
<p><img alt="" src="https://s3-us-west-2.amazonaws.com/oww-files-public/a/ad/250px-Polysorbate_20.png">
Tween (full name Tween 20 or Polysorbate 20) is a common detergent used in biology used in cell lysis and membrane protein solubilisation among other uses.</p>
<h3 id="Triton-X-100">Triton X-100<a class="anchor-link" href="#Triton-X-100">¶</a>
</h3>
<p><img alt="" src="https://s3-us-west-2.amazonaws.com/oww-files-public/3/3d/200px-Triton_X-100.png">
Triton (full name Triton X-100) is a common, nonionic detergent used for solubilisation, permeabilisation, and lysis.</p>
<h3 id="SDS">SDS<a class="anchor-link" href="#SDS">¶</a>
</h3>
<p><img alt="" src="https://s3-us-west-2.amazonaws.com/oww-files-thumb/f/f6/Sodium_dodecyl_sulfate.png/250px-Sodium_dodecyl_sulfate.png">
SDS (sodium dodecyl sulfate/sulphate) is an anionic detergent effective in both acidic and alkaline solutions. SDS has a wide variety of applications, but is most often used in protein and lipid solubilisation.</p>
<h4 id="Warnings">Warnings<a class="anchor-link" href="#Warnings">¶</a>
</h4>
<ul>
<li>highly flammable</li>
<li>harmful in contact with skin and if swallowed</li>
<li>irritating to eyes, respiratory system and skin</li>
</ul>
<hr>
</div>
</div>
</div>
    </div>
    <aside class="postpromonav"><nav><ul itemprop="keywords" class="tags">
<li><a class="tag p-category" href="../../categories/materials/" rel="tag">materials</a></li>
            <li><a class="tag p-category" href="../../categories/protocols/" rel="tag">protocols</a></li>
        </ul>
<ul class="pager hidden-print">
<li class="previous">
                <a href="../python-model-fitting/" rel="prev" title="Model fitting by python">Previous post</a>
            </li>
            <li class="next">
                <a href="../ANOVA-python/" rel="next" title="파이썬으로 분산분석(ANOVA)">Next post</a>
            </li>
        </ul></nav></aside><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_HTMLorMML" integrity="sha384-3lJUsx1TJHt7BA4udB5KPnDrlkO8T6J6v/op7ui0BbCjvZ9WqV4Xm6DTP6kQ/iBH" crossorigin="anonymous"></script><script type="text/x-mathjax-config">
 MathJax.Hub.Config({
     tex2jax: {
         inlineMath: [ ['$','$'], ["\\(","\\)"] ],
         displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
         processEscapes: true
     },
     displayAlign: 'center', // Change this to 'left' if you want left-aligned equations.
     "HTML-CSS": {
         styles: {'.MathJax_Display': {"margin": 0}}
     }
 });
 </script></article><!--End of body content--><footer id="footer">
            Contents © 2023         <a href="mailto:partrita@gmail.com">Taeyoon Kim</a> - Powered by         <a href="https://getnikola.com" rel="nofollow">Nikola</a>         
            
            
        </footer>
</div>
</div>


            <script src="../../assets/js/jquery.min.js"></script><script src="../../assets/js/popper.min.js"></script><script src="../../assets/js/bootstrap.min.js"></script><script src="../../assets/js/baguetteBox.min.js"></script><script>
    baguetteBox.run('div#content', {
        ignoreClass: 'islink',
        captions: function(element){var i=element.getElementsByTagName('img')[0];return i===undefined?'':i.alt;}});
    </script><!-- Modal --><div id="search-results" class="modal fade" role="dialog" style="height: 80%;">
  <div class="modal-dialog">

    <!-- Modal content-->
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close" data-dismiss="modal">×</button>
        <h4 class="modal-title">Search Results:</h4>
      </div>
      <div class="modal-body" id="tipue_search_content" style="max-height: 600px; overflow-y: auto;">
      </div>
      <div class="modal-footer">
        <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
      </div>
    </div>
  </div>
</div>
<script>
var siteUrl = "https://partrita.github.io/"
$(document).ready(function() {
    $.when(
        $.getScript( siteUrl + "/assets/js/tipuesearch_set.js" ),
        $.getScript( siteUrl + "/assets/js/tipuesearch.js" ),
        $.Deferred(function( deferred ){
            $( deferred.resolve );
        })
    ).done(function() {
        $('#tipue_search_input').tipuesearch({
            'mode': 'json',
            'contentLocation': siteUrl + '/assets/js/tipuesearch_content.json'
        });
        $('#tipue_search_input').keyup(function (e) {
            if (e.keyCode == 13) {
                $('#search-results').modal()
            }
        });
    });
});
</script>
</body>
</html>
